api/turing/config/config.go

```diff
@@ -263,8 +263,9 @@ type KubernetesLabelConfigs struct {
 // Knative services
 type KnativeServiceDefaults struct {
 	QueueProxyResourcePercentage          int
-	UserContainerCPULimitRequestFactor    float64 `json:"userContainerLimitCPURequestFactor"`
-	UserContainerMemoryLimitRequestFactor float64 `json:"userContainerLimitMemoryRequestFactor"`
+	UserContainerCPULimitRequestFactor    float64
+	UserContainerMemoryLimitRequestFactor float64
+	DefaultEnvVarsWithoutCPULimits        []corev1.EnvVar
```

Comment:
I have no idea why we needed the `json` tags in the first place (these are the API server-level configs and aren't converted to json anywhere), so I'm removing them.